.eslintrc-code-compatibility.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
```

Comment:
Did you also mean to add https://github.com/adobecom/milo/blob/main/.github/workflows/code-compatibility.yaml?